packages/teams-js/src/internal/validOrigins.ts

```diff
@@ -34,7 +35,11 @@ async function getValidOriginsListFromCDN(): Promise<string[]> {
         });
       })
       .catch((e) => {
-        validateOriginLogger('validOrigins fetch call to CDN failed with error: %s. Defaulting to fallback list', e);
+        if (e.name === 'TimeoutError') {
+          validateOriginLogger('validOrigins fetch call to CDN failed due to Timeout. Defaulting to fallback list', e);
+        } else {
```

Comment:
Can you add new unit test(s) to `validOrigins.spec.ts` that validate this new timeout behavior?

Reply:
Hmm I might need to think about this more, but because we are already mocking the `fetch` call since `jest` doesn't have access to that function as it is a Browser API. 

Wouldn't mocking `AbortSignal.timeout` at the same time introduce uncertainty in the validity of the test, since we are now relying on two mocks working correctly together to yield a result that may not be indicative of the real world scenarios?

I could instead have the fetch mock reject with a 'TimeoutError' to ensure that we enter the timeout error logic.